packages/itwinui-css/src/menu/menu.scss

```diff
@@ -100,6 +100,26 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
   }
 
   &.iui-active {
+    @supports (selector(.iui-active:has(+ .iui-active))) {
+      // Styles active items between the first & last
+      + .iui-active {
+        border-radius: 0;
+      }
+
+      // Styles first active item with an active item following it
+      &:has(+ .iui-active) {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+      }
+
+      // Styles last active item
+      &:has(+ :not(.iui-active)),
+      &:last-child {
```

Comment:
Just wondering, is the `:last-child` selector needed? Is the `&:has(+ :not(.iui-active))` not enough?

Reply:
Changes from below made the `:last-child` no longer necessary.